samcli/lib/package/stream_cursor_utils.py

```diff
@@ -16,16 +16,49 @@
         pass
 
 
-def cursor_up(count=1):
-    return ESC + str(count) + "A"
+class CursorFormatter:
+    """
+    Base class for defining how cursor is to be manipulated.
+    """
+
+    def __init__(self):
+        pass
+
+    def cursor_format(self, count):
+        pass
+
+
+class CursorUpFormatter(CursorFormatter):
+    """
+    Class for formatting and outputting moving the cursor up within the stream of bytes.
+    """
+
+    def cursor_format(self, count=0):
+        return ESC + str(count) + "A"
+
+
+class CursorDownFormatter(CursorFormatter):
```

Comment:
why is each termcode modeled as a class?

Reply:
Worth looking into through a refactor.